pkgs/development/python-modules/appthreat-vulnerability-db/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, appdirs
+, tabulate
+, msgpack
+, orjson
+, semver
+, packageurl-python
+, pythonOlder
+, pytestCheckHook
+}:
+
+buildPythonPackage {
```

Comment:
This will make it possible to reference `version`:
```suggestion
buildPythonPackage rec {
```

Reply:
Ups, that was a rookie mistake. Thanks.